third-party/resources/from-vscode/dark/feedback.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@awschristou @justinmk3 Is it worth considering bumping our VS Code version so we can use the [official codicon](https://code.visualstudio.com/updates/v1_42#_codicons-for-reuse) instead of importing the SVG?

Feedback codicon can be found here: https://microsoft.github.io/vscode-codicons/dist/codicon.html

Reply:
Can we schedule it for later? We should be in the habit of not bumping min-version for "nice to have" stuff.

Reply:
I agree with @justinmk3 on this one. We have the luxury on this one of waiting for more ideal adaption of a certain VS Code version before bumping the minver.

Reply:
That's fine, mind if I cut an issue to reevaluate the icons?